sql_server/pyodbc/compiler.py

```diff
@@ -272,8 +274,10 @@ def as_sql(self, qn=None):
         self._wrap_aggregates()
         return super(SQLAggregateCompiler, self).as_sql(qn=qn)
 
-class SQLDateCompiler(compiler.SQLDateCompiler, SQLCompiler):
-    pass
+if DjangoVersion[:2] >= (1,8):
```

Comment:
Unless I'm not understanding the comment right, but shouldn't this conditional be reversed if these classes aren't needed in Django 1.8+?